providers/github/resources/discovery.go

```diff
@@ -159,13 +160,23 @@ func org(runtime *plugin.Runtime, orgName string, conn *connection.GithubConnect
 			if user.Name.Data == "" {
 				continue
 			}
+			labels := map[string]string{}
+			for j := range org.GetCustomProperties().Data {
+				customProperty := org.GetCustomProperties().Data[j].(*mqlGithubOrganizationCustomProperty)
+				value := ""
+				if customProperty.DefaultValue.IsSet() {
+					// if the default value of the org-leve custom property is set, use it as the label value
+					value = customProperty.DefaultValue.Data
+				}
+				labels[customProperty.Name.Data] = value
+			}
 			cfg := conf.Clone(inventory.WithoutDiscovery(), inventory.WithParentConnectionId(conn.ID()))
 			cfg.Options["user"] = user.Login.Data
 			assetList = append(assetList, &inventory.Asset{
 				PlatformIds: []string{connection.NewGithubUserIdentifier(user.Login.Data)},
 				Name:        user.Name.Data,
 				Platform:    connection.NewGithubUserPlatform(user.Login.Data),
-				Labels:      make(map[string]string),
+				Labels:      labels,
```

Comment:
We need to implement this for `repo` as well, see https://github.com/mondoohq/cnquery/pull/5211/files#diff-9af16f012a12b647b30c4cd9d5e009d39ae5bc7af5fa70969aa6407576c6e5daR209

When you call `cnquery shell github hit-training/test-nest-iter2` the labels are not fetched and gathered

```
cnquery> github.repository.customProperties
github.repository.customProperties: {
  Owner: "john@doo.com"
}

cnquery> asset.labels
asset.labels: {}
```

Reply:
I also tested:

```
±> cnquery shell github org hit-training --repos test-nest-iter2                                                                    
cnquery> github.repository.customProperties
github.repository.customProperties: {}
cnquery> asset.labels
asset.labels: {}
```

Reply:
Oh nice catch! I think I made a mistake and added them to members/users instead of the organization 🤦🏽‍♂️

I am moving them now!